sdk/identity/identity/test/node/msalNodeTestSetup.ts

```diff
@@ -59,9 +59,9 @@ export async function msalNodeTestSetup(
         AZURE_CLIENT_SECRET: "azure_client_secret",
         AZURE_USERNAME: "azure_username",
```

Comment:
do we need two separate environment variables for username & password?

Reply:
Good catch, let me check and confirm - I'll delete them if they're not actually being used